docs/tutorial/visualization/visualizer_advanced.rst

```diff
@@ -0,0 +1,367 @@
+.. _visualizer_advanced:
+
+Advanced Open3D visualizer
+==========================
+
+Introduction
+---------------
+
+.. epigraph:: Open3D provides a convenient function for visualizing geometric objects: ``draw()``. The ``draw()`` function allows you to visualize multiple geometry objects *(PointClouds, LineSets, TriangleMesh)* and images together along with optional, high-quality, physically based (PBR) materials. This tutorial covers more advanced usages of  ``draw()`` calls. For basic ``draw()`` usage, please see the Basic :doc:`visualizer_basic` tutorial.
+
+
+Getting started
+---------------
+
+.. tip::
+    This **Getting started** section applies to all subsequent examples below
+	 
+1. Open your preferred Python environment;
+2. In Python, set up common imports that we will use throughout this tutorial:
+
+.. code-block:: python
+
+    >>> import open3d as o3d
+    >>> import open3d.visualization as vis
+    >>> import numpy as np
+
+
+Advanced examples
+-----------------
+
+In the Basic :doc:`visualizer_basic` tutorial, we covered how to render ``Tensor`` and ``TriangleMesh`` shapes, raster models, and how to control their display programmatically via code and interactively by using Open3D Visualizer UI. This section expounds on those topics to cover more advanced visualization techniques.
+
+
+.. _rendering_models:
+
+Rendering models
+::::::::::::::::
+
+Rendering ``TriangleMesh``'es of 3D models
+""""""""""""""""""""""""""""""""""""""""""
+
+In the Basic :doc:`visualizer_basic` tutorial, we showed how to use Open3D datasets. In this tutorial, we will likewise be using Open3D datasets to load 3D models.
+
+.. seealso::
+
+    For more information on datasets, please refer to the 
+    :doc:`Open3D Datasets page <../data/index>`
+
+We also demonstrated how to apply materials manually to built-in Open3D geometries. It is also possible to load ``TriangleMesh``'es from full 3D models using the ``o3d.io.read_triangle_mesh()`` method, as you will see below: 
+
+
+.. code-block:: python
+
+    # Initialize the monkey dataset with downloaded and extracted 3D model
+    >>> monkey_model = o3d.data.MonkeyModel()
+    [Open3D INFO] Downloading https://github.com/isl-org/open3d_downloads/releases/download/20220301-data/MonkeyModel.zip
+    [Open3D INFO] Downloaded to /home/intel/open3d_data/download/MonkeyModel/MonkeyModel.zip
+    [Open3D INFO] Extracting /home/intel/open3d_data/download/MonkeyModel/MonkeyModel.zip.
+    [Open3D INFO] Extracted to /home/intel/open3d_data/extract/MonkeyModel.
```

Comment:
```
    [Open3D INFO] Downloading https://github.com/isl-org/open3d_downloads/releases/download/20220301-data/MonkeyModel.zip
    [Open3D INFO] Downloaded to /home/intel/open3d_data/download/MonkeyModel/MonkeyModel.zip
    [Open3D INFO] Extracting /home/intel/open3d_data/download/MonkeyModel/MonkeyModel.zip.
    [Open3D INFO] Extracted to /home/intel/open3d_data/extract/MonkeyModel.
```

We may remove these console logs for `Downloading and Extracting`. Not that relevant. Same for the rest of the file.

Reply:
Thanks. Will clean it up.

Reply:
Done.